### PR TITLE
[docs] Add status header for tanzuobservability exporter

### PR DESCRIPTION
**Pull request by Frapschen**
**Description:** <Describe what has changed.>
Add status header for tanzuobservability exporter

### CODE REVIEW

**Review by TylerHelmuth — APPROVED** [review]
/cc @oppegard @thepeterstone @keep94 as code owners to verify beta stability.

**Comment by TylerHelmuth** [comment]
> @TylerHelmuth is this status header something that other exporters are adopting? I haven't seen it before.

Yes, a new standard being tracked here: #10116

File: exporter/tanzuobservabilityexporter/README.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
 # Tanzu Observability (Wavefront) Exporter
 
+| Status                   |                |
+| ------------------------ |----------------|
+| Stability                | [beta]         |
+| Supported pipeline types | trace, metrics |
```
Comment (TylerHelmuth) [inline exporter/tanzuobservabilityexporter/README.md:6]:
```suggestion
| Supported pipeline types | traces, metrics |
```

**Review by oppegard — APPROVED** [review]
LGTM

**Review by codeboten — APPROVED** [review]

**Pull request merged**